File: device-sdk/src/main/java/io/oddworks/device/exception/OddParseException.java
package io.oddworks.device.exception;

public class OddParseException extends RuntimeException {
    public OddParseException() {
        super();
    }

    public OddParseException(String detailMessage) {
        super(detailMessage);
    }

    public OddParseException(String detailMessage, Throwable throwable) {
        super(detailMessage, throwable);
    }

    public OddParseException(Throwable throwable) {
        super(throwable);
    }
}

File: device-sdk/src/main/java/io/oddworks/device/exception/OddParseException.kt
package io.oddworks.device.exception

class OddParseException : RuntimeException {
    constructor() : super() {
    }

    constructor(detailMessage: String) : super(detailMessage) {
    }

    constructor(detailMessage: String, throwable: Throwable) : super(detailMessage, throwable) {
    }

    constructor(throwable: Throwable) : super(throwable) {
    }
}
